via/views/view_video.py

```diff
@@ -38,11 +57,34 @@ def youtube(request, url, **kwargs):
         "api": {
             "transcript": {
                 "doc": "Get the transcript of the current video",
-                "url": request.route_url("api.youtube.transcript", video_id=video_id),
+                "url": request.route_url(
+                    "api.youtube.transcript",
+                    video_id=video_id,
+                    transcript_id=_get_transcript_id(
+                        youtube_service, video_url, via_config
+                    ),
+                ),
                 "method": "GET",
                 "headers": {
                     "Authorization": f"Bearer {ViaSecurityPolicy.encode_jwt(request)}"
                 },
             }
         },
     }
+
+
+def _get_transcript_id(youtube_service, url, via_config):
+    # Try the `via.video.lang` param first
+    if transcript_id := via_config.get("video", {}).get("lang"):
+        return transcript_id
```

Comment:
```suggestion
```

We should remove this as it'll be dead code: nothing is every going to be passing the `via.video.lang` param. We can add support for this query param back in a later PR when we need it